frontend/pyproject.toml

```
@@ -0,0 +1,7 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.isort]
+profile = "black"
+src_paths = ["isort", "test"]
```

```suggestion
src_paths = ["app"]
```

Misses `setup.py` though.